handlers.js

```diff
@@ -255,7 +255,7 @@ module.exports = function(port) {
             identityCheckParams = assign({ip: (port.bus.config.debug && request.headers['x-forwarded-for'])
```

Comment:
then please change the order of assignment

in future 'x-forwarded-for' should also be possible to be enabled in production, through configuration, as servers often are behind firewalls that will set this header